src/Backend/Modules/MediaLibrary/Resources/config/console.yml

```diff
@@ -1,10 +1,9 @@
 services:
-    media_library.console.delete_media_items:
-        class: Backend\Modules\MediaLibrary\Console\MediaItemDeleteAllCommand
+    media_library.console.cache_clear:
```

Comment:
forkcms.console.cache.clear add it to this one
and add something like this http://symfony.com/doc/current/console/calling_commands.html